File: models/transacao.py

from datetime import datetime

class Transacao:
    def __init__(self, correlation_id, data_hora, conta_origem, conta_destino, valor):
        self.correlation_id = correlation_id
        self.data_hora = datetime.strptime(data_hora, "%d/%m/%Y %H:%M:%S")
        self.conta_origem = conta_origem
        self.conta_destino = conta_destino
        self.valor = valor

File: models/Transacao.js

class Transacao {
    constructor(correlationId, dataHora, contaOrigem, contaDestino, valor) {
        this.correlationId = correlationId;
        this.dataHora = new Date(dataHora);
        this.contaOrigem = contaOrigem;
        this.contaDestino = contaDestino;
        this.valor = valor;
    }
}

module.exports = Transacao;
